app/scripts/presenters/visit_presenter.js

```diff
@@ -0,0 +1,102 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'underscore'
+], function (_) {
+  'use strict';
+
+  var MINIMUM_IMAGE_ENTROPY = 1.75;
+  var MINIMUM_IMAGE_WIDTH = 270;
+  var EXTRACTED_IMAGE_DENY_PATTERN = /github.com|(\..{2,3}\/$)|([&?]q=)|([&?]page=\d+)/;
```

Comment:
Not sure if you want to leave a quick comment for future travelers (or mere QE) who may be scratching their heads over all the OR statements in the reg' expression (and why github seems to be special).

![](https://31.media.tumblr.com/ae6d4f98920c1109de72262024373b5a/tumblr_inline_niimdkuGUX1r79k32.gif)